app/src/main/java/com/example/thecocktailsapp/model/domain/Drink.kt

```diff
@@ -0,0 +1,79 @@
+package com.example.thecocktailsapp.model.domain
+
+import com.example.thecocktailsapp.model.CocktailIInfo.CocktailsResponse
+
+
+data class Drink(
+    val idDrink: String?,
+    val alcoholic: String?,
+    val category: String?,
+    val drinkName: String?,
+    val drinkThumb: String?,
+    val glass: String?,
+    val ingredients: List<String?>?,
+    val instructions: String?,
+    val measures: List<String?>?
+)
+
+fun CocktailsResponse?.mapToDrinkDomain(): Drink{
```

Comment:
in the model it is not called "mapToDrinkDomain". i believe that we need to change it . Right?

Reply:
Ok, I'll change it